src/components/Supplier/AppSupplierData/AddSupplierData.tsx

```diff
@@ -1,17 +1,213 @@
-import { Flex, Box } from '@chakra-ui/react';
-import AddSupplierDataName from './AddSupplierDataName';
-import AddSupplierDataAddress from './AddSupplierDataAddress';
+import {
+  Flex,
+  Box,
+  Container,
+  Grid,
+  Input,
+  Textarea,
+  NumberInput,
+  NumberInputField,
+  VStack,
+  StackDivider,
+} from '@chakra-ui/react';
+import { useForm } from 'react-hook-form';
+import AddSupplierButtons from 'components/Supplier/AddSupplierButtons';
+import styled from 'styled-components';
 
-const AddSupplierData = () => (
-  <Flex>
-    <Box w="80em" h="26em">
-      <AddSupplierDataName />
-    </Box>
-    <Box w="6em" h="26em" />
-    <Box w="100em" h="26em">
-      <AddSupplierDataAddress />
-    </Box>
-  </Flex>
-);
+const AddSupplierData = () => {
+  const { register, handleSubmit, errors } = useForm();
+
+  const StylesS = styled.span`
+    color: red;
+    padding: 0.5rem 0 0 0;
+  `;
+
+  const onSubmit = (data) => {
```

Comment:
`data` should be typed, even with temporal interface / type.

Reply:
You are right. I forgot to declare the type `data`

Reply:
@kacpergadula I meant creating an interface that mirrors the data on the form, but for now it can stay that way.